src/applications/gi/components/vet-tec/EybVetTecCalculatorForm.jsx

```diff
@@ -0,0 +1,191 @@
+import PropTypes from 'prop-types';
```

Comment:
the corresponding non VET TEC version of this is `src/applications/gi/components/profile/EstimateYourBenefitsForm.jsx` might be best to use rename this to `src/applications/gi/components/vet-tec/EstimateYourBenefitsForm.jsx`

Reply:
Right now this file should be in `containers` because it's connected to the Redux store, but instead of moving it, can you have the parent pass in `selectedProgram` since that's the one thing `mapStateToProps` is being used for?

Reply:
Done.